app/src/main/java/ru/chaichuk/hwapp/api_v3/dto/ConfigDTO.kt

```diff
@@ -0,0 +1,36 @@
+package ru.chaichuk.hwapp.api_v3.dto
+
+import kotlinx.serialization.*
+
+@Serializable
+data class ConfigDTO (
```

Comment:
it's optional because it's a tiny project
but real production projects don't create in one file so much pojos

Reply:
I used quicktype.io to create that classes - it creates one file for one response. Next time I will split that classes in files